### PR TITLE
chore: introduce website docker image build and push workflow

### PR DESCRIPTION
**Pull request by ali-tog**
The website project will use this work flow

- https://github.com/toggleglobal/website

### CODE REVIEW

File: .github/workflows/image-nextjs.yml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            SENTRY_AUTH_TOKEN=${{  secrets.SENTRY_AUTH_TOKEN }}
+            NEXT_PUBLIC_GA=${{ secrets.NEXT_PUBLIC_GA }}
+            NEXT_PUBLIC_NAVER=${{  secrets.NEXT_PUBLIC_NAVER }}
+            SIB_KEY=${{ secrets.SIB_KEY }}
+            NEXT_PUBLIC_DOMAIN=${{ secrets.NEXT_PUBLIC_DOMAIN }}
+            CONTACT_EMAIL=${{  secrets.CONTACT_EMAIL }}
+            NEXT_PUBLIC_API_PATH=${{ secrets.NEXT_PUBLIC_API_PATH }}
+            STRAPI_PREVIEW_SECRET=${{ secrets.STRAPI_PREVIEW_SECRET }}
+            NEXT_PUBLIC_COOKIE_PRO=${{ secrets.NEXT_PUBLIC_COOKIE_PRO }}
+            NEXT_PUBLIC_EARNINGS_URL=${{ secrets.NEXT_PUBLIC_EARNINGS_URL }}
```
Comment (ali-tog) [inline .github/workflows/image-nextjs.yml:88]: @luisprates those secrets are set on website repo, would they be available for this workflow when called from website

Comment (luisprates) [reply .github/workflows/image-nextjs.yml:88]: All these, except for SENTRY_AUTH_TOKEN and SIB_KEY, are defined as variables in the website repository. Therefore they need to be called as this` vars. NEXT_PUBLIC_DOMAIN `. Contrary to secrets that can be set as inherit, variables must be passed as arguments from a workflow to a reusable workflow. I'll provide the necessary changes.

**Comment by luisprates** [comment]
@ali-tog  we can delete this

**Comment by ali-tog** [comment]
> @ali-tog we can delete this

@luisprates yes please , i tried drop this previously but i don't think i have authority todo so

**Pull request closed without merging**